net-gateway/src/main.rs

```diff
@@ -15,12 +15,12 @@ async fn main() -> std::io::Result<()> {
             .wrap(
                 Cors::default()
                     // for profuction use this will be changed
-                    .allowed_origin("http://localhost:4000")
+                    .allowed_origin("http://localhost:3000")
```

Comment:
Why exactly have you changed the port?

Reply:
Our new react project usually runs on 3000 port